moveit_experimental/CMakeLists.txt

```diff
@@ -56,11 +57,10 @@ catkin_package(
     pluginlib
   DEPENDS
     Boost
-    Eigen3
```

Comment:
I may have missed some context, but shouldn't the `Eigen` dependency be exported here? Information in `package.xml` is not used when generating the `PkgConfig` files, only the `catkin_package(..)` function does that.